File: 0771-jewels-and-stones/0771-jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    jewels=new Set(jewels)
    let count=0
    for(let i of stones){
        if(jewels.has(i)) count++
    }
    return count
};

File: 0771-jewels-and-stones/0771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        unordered_set <int> jewelsSet(jewels.begin(),jewels.end());
        int count=0;
        for(char i : stones){
            if(jewelsSet.count(i)){
                count++;
            }
        }
        return count;
    }
};